packages/framework-integration-tests/integration/providers/aws/subscriptions.integration.ts

```diff
@@ -1,12 +1,19 @@
-import { countSubscriptionsItems, DisconnectableApolloClient, graphQLClientWithSubscriptions, waitForIt } from './utils'
+import {
+  countConnectionsItems,
+  countSubscriptionsItems,
+  DisconnectableApolloClient,
+  graphQLClientWithSubscriptions,
+  waitForIt,
+} from './utils'
 import { random } from 'faker'
 import gql from 'graphql-tag'
+import { expect } from 'chai'
 import * as chai from 'chai'
 import { Observable } from 'apollo-client/util/Observable'
 
 chai.use(require('chai-as-promised'))
 
-describe('subscriptions', () => {
+describe.only('subscriptions', () => {
```

Comment:
What does this mean?

Reply:
Woah!! This passed unnoticed, I will remove it. 

If you put this, when you run the test suite, only this test will be executed. It is ideal for developing but really dangerous.

Reply:
Done: 170da5782af35d0fea0527c1b4cba14aee93bbbb